docs/file-reference.md

```diff
@@ -161,3 +163,19 @@ Default value for type is `internal`.
 - `internal` - port will be accessible only from inside the Kubernetes cluster
 - `external` - port will be accessible from inside and outside of the Kubernetes cluster
 
+#### host 
+| type        | required | possible values                                           |
+|-------------|----------|-----------------------------------------------------------|
+| string      |    no    | FQDN (fully qualified domain name) as defined by RFC 3986 |
+
+Specifying host will make your application accessible outside of the cluster on domain specified as a value of `host`. (Note: this requires you to manually setup DNS records to point to your cluster's Ingress router. For development you can use services like http://nip.io/ or [http://xip.io/].)
+
+#### path 
```

Comment:
I thin we need a little bit more explanation what path does. Ideally with some examples.